web/src/pm_list.js

```diff
@@ -194,6 +215,9 @@ function zoom_in() {
     $(".private_messages_container").removeClass("zoom-out").addClass("zoom-in");
     $("#streams_list").hide();
     $(".left-sidebar .right-sidebar-items").hide();
+
+    // Add filter section for direct messages.
+    $(".private_messages_container").children("div.pm_filter_row").append(render_filter_pms());
```

Comment:
I don't think we need the `children` query here, we can do this in one single query.

Reply:
Updated!